tests/Animation/AnimationTest.php

```diff
@@ -103,10 +106,13 @@ public function it_can_exit_to_top()
 
         $this->exitTopFrame6();
 
-        $this->cli->animation('404', $this->getSleeper(11))->exitTo('top');
+        $x = $this->cli->animation('404', $this->getSleeper(11))->exitTo('top');
```

Comment:
```suggestion
        $this->cli->animation('404', $this->getSleeper(11))->exitTo('top');
```

Reply:
Fixed -- artifact code -- I was trying to see if this returned a value I could perform an assert on.